src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidates.tsx

```diff
@@ -134,37 +140,52 @@ class Candidates extends React.Component<Props, State> {
   }
 
   getFilterOptions(candidates: Image['candidates']) {
-    return {
-      [t('Status')]: [
-        ...new Set(candidates.map(candidate => candidate.download.status)),
-      ].map(status => ({
+    const {imageStatus} = this.props;
+
+    const filterOptions = {};
+
+    const candidateStatus = [
+      ...new Set(candidates.map(candidate => candidate.download.status)),
+    ];
+
+    if (candidateStatus.length > 1) {
+      filterOptions[filterOptionCategories.status] = candidateStatus.map(status => ({
```

Comment:
Are the keys to `filterOptions` also translated strings?

Reply:
yes they are

Reply:
yes https://github.com/getsentry/sentry/blob/5bb78cf4421b8259deb99efb14e0c3eb0178b8ce/src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/debugImageDetails/candidates.tsx#L24